packages/squiggle-lang/src/rescript/Distributions/PointSetDist/Mixed.res

```diff
@@ -359,14 +357,10 @@ let combinePointwise = (
   t2: t,
 ): result<t, 'e> => {
   let reducedDiscrete =
-    [t1, t2]
-    |> E.A.fmap(toDiscrete)
-    |> E.A.O.concatSomes
-    |> Discrete.reduce(~integralSumCachesFn, fn)
+    [t1, t2]->E.A.fmap(toDiscrete) |> E.A.O.concatSomes |> Discrete.reduce(~integralSumCachesFn, fn)
```

Comment:
why not a "->" before concatSomes?

Reply:
Easier to just look at the immediately surrounding context when I was doing many replacements, but I do have this fixed in a later commit.